automation/docker/application.conf.ctmpl

```diff
@@ -48,6 +61,17 @@ users {
   luna = "luna.temp@{{template "USER_DOMAIN"}}"
   lunaSubjectId = "{{if or (eq $environment "dev") (eq $environment "alpha")}}111010567286567716739{{else if eq $environment "qa"}}117891551413045861932{{else}}[undefined]{{end}}"
   neville = "neville.temp@{{template "USER_DOMAIN"}}"
+
+  # for smoketests
+  smoketestpassword = {{if eq $environment "prod"}}"{{$users.Data.users_passwd}}"{{else}}"alohomora"{{end}}
```

Comment:
Password needs to be read from vault. It should always be there no matter what `$environment` is.